src/main/java/com/puppycrawl/tools/checkstyle/gui/Main.java

```diff
@@ -47,6 +49,8 @@ public static void main(final String... args) {
                 final File sourceFile = new File(args[0]);
                 mainFrame.openFile(sourceFile);
             }
+            final URL iconUrl = Main.class.getResource("icon.png");
```

Comment:
all images are at https://github.com/checkstyle/resources/tree/master/img/cs-logos-twitter-gplus-backgrounds and in other folders of this repo.
https://github.com/checkstyle/resources/blob/master/img/cs-logos-twitter-gplus-backgrounds/cs-120x120.png is better as CS and pancil are in same size.


and we might figureout from where you got this page and fix it also

Reply:
done